src/helpers/objectsPath.ts

```diff
@@ -26,7 +26,24 @@ const getPythonObjects = async (): Promise<string[]> => {
   }
 };
 
+const getNodeJsObjects = async (): Promise<string[]> => {
+  let soOutput = "";
+  try {
+    await exec("node", ["-e", "console.log(process.execPath);"], {
```

Comment:
This will mean that all the calls to global Node functions will never be displayed in the call graph?
I thought we decided to keep them, eg to see calls like `new Date()`